_docs/_latest/configure/email-notification.md

```diff
@@ -18,7 +20,7 @@ process below:
 1.  [Sign up for a SendGrid account](https://sendgrid.com/).
 1.  Create a general
     [API Key](https://sendgrid.com/docs/User_Guide/Settings/api_keys.html).
-1.  Edit the following in `forseti_conf.yaml`:
+1.  Edit the following in `forseti_conf_server.yaml`:
```

Comment:
Since you're describing configuration files here, can you use the format where we describe the type, possible values, and a description?